ci/pipeline-release-deploy.yml

```diff
@@ -0,0 +1,59 @@
+############################
+#  SHARED
+
+cf-image: &cf-image
+  platform: linux
+  image_resource:
+    type: registry-image
+    source:
+      aws_access_key_id: ((ecr-aws-key))
+      aws_secret_access_key: ((ecr-aws-secret))
+      repository: harden-concourse-task
+      aws_region: us-gov-west-1
+      tag: ((harden-concourse-task-tag))
+
+jobs:
+- name: configure-pipeline
+  plan:
+  - get: src
+    trigger: true
+    params: {depth: 1}
+  - set_pipeline: self
+    file: src/ci/pipeline-release-deploy.yml
+    instance_vars:
+        deploy-env: release
+        git-branch: ((git-branch))
+        product: ((product))
+
+- name: update-release-branch
+  plan:
+  - get: src
+    trigger: true
+  - task: update-release-branch
+    config:
+      <<: *cf-image
+      inputs:
+        - name: src
+      params:
+        GH_EMAIL: ((pages-operations-github-user-info.email))
+        GH_USERNAME: ((pages-operations-github-user-info.username))
+        GH_BOT_GPG_KEY: ((pages-operations-github-user-gpg.private_key))
+        GH_BOT_SSH_KEY: ((pages-gpg-operations-github-sshkey.private_key))
+        GH_BOT_GPG_TRUST: ((pages-operations-github-user-gpg-trust))
+        GH_TOKEN: ((pages-operations-ci-github-token))
+      run:
+        dir: src
+        path: ci/tasks/update-release-branch.sh
+
+############################
+#  RESOURCES
+
+resources:
+  - name: src
+    type: git
+    icon: github
+    source:
+      uri: git@github.com:cloud-gov/pages-core.git
+      branch: ((git-branch))
+      commit_verification_keys: ((cloud-gov-pages-gpg-keys))
```

Comment:
Is there any documentation or a runbook on best practices to set or update certificates in credhub?

Reply:
I got personal support from @soutenniza but agree that this could use a runbook somewhere

Reply:
Sounds good. I'll make an issue for the platform squad to put together a runbook for it.